### ISSUE TITLE
Can the scoreboard show cumulative scores?

### ISSUE THREAD

**Issue by flackr** [ArnoldSmith86/virtualtabletop#1537]
For many games, e.g. scrabble, it's common to have the score for each round be the player's cumulative score rather than what they scored on that round. AFAICT, while you can do this by scoring the current round scores you then can't use the totals column because it would total those numbers.

E.g. what I would want is something like this (Where "Final" is the totals row):

| Round | Player 1 | Player 2 |
|---|---|---|
| 1 | 5 | 9 |
| 2 | 12 | 14 |
| 3 | 24 | 22 |
| Final | 24 | 22 |

**Comment by 96LawDawg** [comment]
So just hide the totals row?  If the last score in the box is the cumulative score, why do you need to repeat it?

But you are correct.  What you want is not possible currently.  There is no way to exclude certain rows from the totals. Didn't even consider the possibility.

**Comment by flackr** [comment]
Well, I want the last row with the current score to always be visible. But maybe that could be accomplished by automatically scrolling it into view instead.

But I still think if it would be better in the long term if we make the scoreboard support current total vs per round scores. Otherwise I also can't use the `_totals` property for example because it will say Player 1 has 41 points.

**Comment by rogerl50** [comment]
The display changes could be done pretty easily without changing much. We'd have to think through what happens when you click to edit, since the scores shown probably don't actually appear in the seat's scores array.

**Comment by bjalder26** [comment]
I think something like this to hide the totals row would be a workaround until a PR is created.

```JSON
  "css": {
    ">.scoreboardIntermediate .totalsLine ": {
      "display": "none"
    }
  }
  ```
![image](https://github.com/user-attachments/assets/15211ad6-b0e8-4af4-9da7-65cb8ac7e19b)

**Comment by 96LawDawg** [comment]
I don't think that is what this issue is about.  While your css solution works, it is much easier to just `"showTotals": false`.  This issue is about just repeating the last row as the total.

**Comment by rogerl50** [comment]
So instead of round-by-round scores, you want cumulative scoring. We could introduce a property saying so, and probably change the default on `showTotals` to `false` in that case. That would not be hard.

But once you do this, the per-round scores dont show up anywhere. How would you want the cell editing to work?
